it/mail/chrome/messenger/cloudfile/management.dtd

```diff
@@ -1,8 +1,8 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this file,
    - You can obtain one at http://mozilla.org/MPL/2.0/.  -->
-<!ENTITY cloudfileMgmt.privacyPolicy "Privacy Policy">
-<!ENTITY cloudfileMgmt.termsOfService "Terms of Service">
-<!ENTITY cloudfileMgmt.usedSpace "Used Space:">
-<!ENTITY cloudfileMgmt.unusedSpace "Unused Space:">
-<!ENTITY cloudfileMgmt.upgradeOffer "Get more space">
+<!ENTITY cloudfileMgmt.privacyPolicy "Politica sulla privacy">
+<!ENTITY cloudfileMgmt.termsOfService "Termini del servizio">
```

Comment:
CORRECTIONS NEEDED: It should be as follows: 

 +<!ENTITY cloudfileMgmt.termsOfService "Termini di servizio">